third_party/blink/renderer/core/script/pending_script.h

```diff
@@ -156,6 +156,9 @@ class CORE_EXPORT PendingScript : public GarbageCollected<PendingScript>,
     return original_execution_context_;
   }
 
+  // IDs for nodes which executing the script depends on.
+  Vector<int> record_replay_dependency_node_ids_;
```

Comment:
Why put replay data structures in non-replay files?
Can we move this please?

I know you are saying that you want to push off "proper code guidelines" until "we have time", but I'd rather, we start moving in the right direction now, instead of waiting for something that will never come. Does that make sense?